cmd/root.go

```diff
@@ -167,6 +167,7 @@ func rootCmd(o *options.Options) error {
 		&repoResult,
 		checkDocs,
 		pol,
+		repoResult.MaintainersAnnotation,
```

Comment:
we already have `repoResult` as an earlier argument. I don't think we need to change the signature here.

Reply:
Done.

Reply:
Thanks for the call out, I forgot to remove it.